src/App.vue

```diff
@@ -146,13 +146,14 @@ export default class App extends Mixins(DateMixin, FilingTemplateMixin, LegalApi
     // Evaluate name request pre conditions
     const nameRequest = await this.evaluateNRPreconditions()
     if (nameRequest && nameRequest.nrNum && nameRequest.isConsumable) {
-      // TODO: Handling different NR Formats
+      // TODO: Handling different NR Formats & accept the EntityType dynamically once Namex provides it.
```

Comment:
I think Namex does provide it.

Is there already a check that this is a "BC" NR? (I know atm there's only Corporation NRs so the current check should be commented out and enabled in the future; but the check should be there.)

Reply:
The only property i could see that is close is the `requestTypeCd` prop, which is sending back an unrecognizable type for our front end. 

I updated to leave the check in Commented out for MVP 👍